backend/services/clubService.js

```
@@ -0,0 +1,279 @@
+
+// Add event to club
+async function addEvent(clubId, eventId) {
+  // since events are not implemented yet, this function will not do anything
+  throw new Error("Events are not implemented yet");
+  try {
+    if (!clubId) {
+      throw new Error("clubId and eventId are required");
+    }
+    if (!eventId) {
+      throw new Error("eventId is required");
+    }
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will throw an exception beforehand


(lint/correctness/noUnreachable)